cpp/daal/src/sycl/select_indexed.h

```diff
@@ -75,6 +75,16 @@ class SelectIndexed
         DAAL_CHECK_STATUS_VAR(convertIndicesToLabels(result.indices, dataLabels, nVectors, nK, labelOffset));
         return services::Status();
     }
+    services::Status selectNearestDistancesAndIndices(const UniversalBuffer & distances, const UniversalBuffer & dataLabels, uint32_t nK,
```

Comment:
It looks like a duplication of the function above

Reply:
Fixed. Please check